frontend/src/app/modules/exclusion-dates/exclusion-dates-picker/exclusion-dates-picker.component.ts

```diff
@@ -0,0 +1,96 @@
+import { Component, OnInit } from '@angular/core';
+import { FormControl, FormGroup, Validators } from '@angular/forms';
+import { MatDialogRef } from '@angular/material/dialog';
+import { BaseComponent } from '@core/base/base.component';
+import { TimeRangeValidator } from '@modules/exclusion-dates/validators/time-range-validator';
+import { hourMinutesRegex } from '@shared/constants/model-validation';
+
+@Component({
+    selector: 'app-exclusion-dates-picker',
+    templateUrl: './exclusion-dates-picker.component.html',
+    styleUrls: ['./exclusion-dates-picker.component.sass'],
+})
+export class ExclusionDatesPickerComponent extends BaseComponent implements OnInit {
+    selected: Date | null;
+
+    timeIndexIdSeed = 1;
+
+    timeControlsIdentifiers: number[] = [0];
+
+    formGroup: FormGroup;
+
+    readonly startRangeIdentifier = '0';
+
+    readonly endRangeIdentifier = '1';
```

Comment:
Why do we need this?

Reply:
One of the previous comments was to remove magic numbers. Then I added these properties in order to give them names. Why do I need them? The reason is that I dynamically add and remove inputs to formgroup and I need to somehow give names for each pair of inputs and have access to two inputs together. I create input name as unique number id + "0" or "1" depending on the type of the input: the input for start time range or end time range. I can replace "0" and "1" with meaningful values such as "startRange" and "endRange"

Reply:
Yes, it woulf be better to use "startRange" and "endRange"

Reply:
Fixed